packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -425,31 +437,25 @@ export default class DCWebxdc extends SplitOut {
         Exclude<Parameters<setPermissionRequestHandler>[0], null>
       >[1]
       const loggedPermissionRequests: { [K in permission_arg]?: true } = {}
-      /** prevents webxdcs from spamming the log */
-      const logPermissionRequest = (permission: permission_arg) => {
-        if (loggedPermissionRequests[permission]) {
-          return
-        }
-        loggedPermissionRequests[permission] = true
-        log.info(
-          `webxdc '${webxdcInfo.name}' requested "${permission}" permission, but we denied it.
-If you think that's a bug and you need that permission, then please open an issue on github`
-        )
-      }
+
       const permission_handler = (permission: permission_arg) => {
-        if (permission == 'pointerLock') {
-          log.info(`allowed webxdc '${webxdcInfo.name}' to lock the pointer`)
-          // because games might lock the pointer
-          return true
-        }
-        if (permission == 'fullscreen') {
-          log.info(`allowed webxdc '${webxdcInfo.name}' to go into fullscreen`)
-          // games might do that too
-          return true
+        const isAllowed: boolean = ALLOWED_PERMISSIONS.includes(permission)
+
+        // Prevent webxdcs from spamming the log
+        if (!loggedPermissionRequests[permission]) {
+          loggedPermissionRequests[permission] = true
+          if (isAllowed) {
+            log.info(
+              `ALLOWED permission '${permission}' to webxdc '${webxdcInfo.name}'`
+            )
+          } else {
+            log.info(
+              `DENIED permission '${permission}' to webxdc '${webxdcInfo.name}'. If you think that's a bug and you need that permission, then please open an issue on github.`
```

Comment:
Note that I'm using this on a single row for a reason. If you add a line break in there, it will mess up the log entry. All rows in the log should start the same, i.e. with the time.